supervisor/api/middleware/security.py

```diff
@@ -208,7 +208,7 @@ async def system_validation(
             CoreState.RUNNING,
             CoreState.FREEZE,
         ):
-            return api_return_error(
+            return await api_return_error(
```

Comment:
I think there happened some merge/rebase issues. `api_return_error` is no longer a co-routine, so no `await` needed here.